### DISCUSSION TITLE
Observations are not being updated

### DISCUSSION THREAD

**Discussion by brewin** [weather-gov/api#738] [General]
Station observations have not been updated in over 4 hours. Every location I’ve checked on [beta.weather.gov](https://beta.weather.gov/point/41.883/-87.632?suggestedPlaceName=) shows last updated 3:53 PM CDT. I have noticed this happening frequently over the last few days while testing the API. Will this issue be resolved soon?

**Comment by dwhitemv25** [comment]
As far as today anyway, it looks like there are big problems in College Park. Other services hosted out of there are down as well, see my reply to #737. Last METARs I got from nearby stations through API are from 20:54Z and my NWWS-OI links went down at 21:28Z. Sanity check on CheckWX shows current METARs, so they're getting sent by the station at least.

**Comment by brewin** [comment]
Going on 14 hours now. Also, there are active alerts for the Chicago area and none of them are showing up. Meanwhile, everything appears correct on forecast.weather.gov. 

Just 2 days until the MapClick service is removed.  Removal of the MapClick service should be postponed.

**Reply by dwhitemv25** [reply]
Looking at overnight ADMSDM messages, the network in CP went down. Work has been ongoing to restore services.

NWWS-OI failed over to Boulder and is back up as of 7:30am PDT / 1430Z.

**Reply by sullynole** [reply]
Once MADIS is restored API observations should be available again.  There are no plans to remove MapClick page from forecast.weather.gov.  If referring to digitalDWML and digitalJSON that SCN was updated to be effective TBD.  https://www.weather.gov/media/notification/pdf_2023_24/scn24-50_removal_of_digital_projects_from_forecast_aaa.pdf

**Reply by brewin** [reply]
I'm referring to the dwml and json services.

**Reply by sullynole** [reply]
While it is best to migrate to api for json and dwml where possible, according to the SCN there is not yet a date when those will be removed from forecast.weather.gov.  Improvements will be made to api.weather.gov endpoints first.

**Reply by brewin** [reply]
The SCN mentions the dwml and json services in the subject, but does not mention them in the text. I assumed that only the removal of digitalDWML and digitalJSON had been postponed.

So none of the MapClick data services will be removed on June 27?
